api/src/integration/java/org/pmiops/workbench/google/DirectoryServiceImplIntegrationTest.java

```diff
@@ -0,0 +1,31 @@
+package org.pmiops.workbench.google;
+
+import java.io.IOException;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertTrue;
+import org.junit.Before;
+import org.junit.Test;
+
+public class DirectoryServiceImplIntegrationTest {
```

Comment:
Worth noting in a comment that this actually calls the GSuite API.

Are you planning on adding tests here later that actually create / delete users? (Does that cost us extra?)

Reply:
If feel like the fact that it is calling the GSuite API is implied since there is no mocking. Yes, creating a user would cost us money. I'm inclined to say that this is enough to de-risk this integration point—once I can use the API, I'm confident I can call different endpoints depending on what I want to do.